Sources/MapboxNavigation/Map.swift

```diff
@@ -0,0 +1,30 @@
+
+import Foundation
+import MapboxCoreMaps
+
+extension Map {
+    func getStyleSourceURLs(_ sourceTypes: [String]) -> [String] {
```

Comment:
Rename to `styleSourceURLs(types:)`. Even if the map SDK’s public API uses strings to represent URLs, we should use `URL`s in any extensions for added type safety.

Reply:
I think that original method naming/signature is not correct as this method returns datasets (e.g. `mapbox.mapbox-streets-v8`), not URLs. I think I'll rename it to something like styleSourceDatasets(_:). It'll keep returning `[String]`.